integration/test_fulltext.py

```diff
@@ -300,13 +301,15 @@ def test_ft_create_and_info(self):
         assert isinstance(stop_words, list), f"stop_words should be list, got: {type(stop_words)}"
         assert set(stop_words) == {"the", "and"}, f"Expected stop_words ['the', 'and'], got: {stop_words}"
         
-        # Validate with_offsets setting
+        # Validate with_offsets setting (only present when text_index_schema exists)
```

Comment:
Why does this test need to be changed? Aren't we validating `idx4` here which does have text fields so we should probable expect that these fields exist?